website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/registration-approaches.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Retroactive public goods funding - Registration approaches
```

Comment:
adding title here should automatically add a title on the top of the page, but here the title is not shown:
<img width="863" alt="image" src="https://github.com/privacy-scaling-explorations/maci/assets/9023842/c6aa9a01-3045-43dc-8541-a98dced7e737">